test/reliability.js

```diff
@@ -158,4 +158,73 @@ describe("Crawler reliability", function() {
 
         localCrawler.start();
     });
+
+    describe("when stopping the crawler", function() {
+
+        it("should not terminate open connections unless asked", function(done) {
+            var localCrawler = Crawler.crawl("http://127.0.0.1:3000/");
```

Comment:
Don't worry, I saw that. Just didn't know you'd already merged the other change! Haha. :)